docker/dropin_env_base_jdk/build_image.sh

```diff
@@ -8,13 +8,13 @@
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-IMAGE_NAME=datarobot/dropin-env-base-jdk
-IMAGE_TAG=debian11-py3.9-jdk11.0.16-drum1.10.7-mlops9.1.3
+IMAGE_NAME=datarobotdev/dropin-env-base-jdk
```

Comment:
Is the `datarobotdev` name by intention?

Reply:
Yes.
They change structure, that images will be first uploaded to `datarobotdev`, and then promoted to `datarobot` by accounts with permissions.